plinth/modules/minetest/__init__.py

```diff
@@ -107,17 +122,34 @@ def disable():
     frontpage.remove_shortcut('minetest')
 
 
-class MinetestServiceView(ServiceView):
-    service_id = managed_services[0]
-    diagnostics_module_name = "minetest"
-    description = description
-    show_status_block = True
-
-
 def diagnose():
     """Run diagnostics and return the results."""
     results = []
 
     results.append(action_utils.diagnose_port_listening(30000, 'udp4'))
 
     return results
+
+
+def get_max_players_value():
+    """Return the current Max Players value."""
+    value = aug.get(AUG_PATH + '/max_players')
+    return value
+
+
+def get_creative_mode_value():
+    """Return the current Creative mode value."""
+    value = aug.get(AUG_PATH + '/creative_mode')
```

Comment:
If it's disable, the returned value will be 'false', but that's different from False. We probably want to compare the return value to the string 'true'.